uber/templates/regform.html

```diff
@@ -181,9 +181,151 @@
+                            <span style="text-align: center; font-size: 120%; font-weight: bold;">
+                                <img src="http://placehold.it/50x50?text=icon" alt=""> + $70
+                            </span>
+                            <br>
+                            <button style="width: 100%;  margin-top: 1em;" class="btn btn-primary">
+                                Supgrade 2
+                            </button>
+                            <ul style="list-style-type: none; margin-top: 1em; padding-left: 0; font-size: 80%">
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    Sweet Swag 2
+                                </li>
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    Is Named 2
+                                </li>
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    In this area 2
+                                </li>
+                            </ul>
+                        </label>
+
+
+                        <label class="btn btn-default" style="text-align: left;">
+                            <input type="radio" name="{}" autocomplete="off" value="80" onchange="donationChanged();" {} />
+                            <h4 style="text-align: center; text-transform: uppercase;">
+                                Test Tier 2
+                            </h4>
+                            <span style="text-align: center; font-size: 120%; font-weight: bold;">
+                                <img src="http://placehold.it/50x50?text=icon" alt=""> + $800
+                            </span>
+                            <br>
+                            <button style="width: 100%;  margin-top: 1em;" class="btn btn-primary">
+                                Supgrade 2
+                            </button>
+                            <ul style="list-style-type: none; margin-top: 1em; padding-left: 0; font-size: 80%">
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    Sweet Swag 2
+                                </li>
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    Is Named 2
+                                </li>
+                                <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                    In this area 2
+                                </li>
+                            </ul>
+                        </label>
+                        <!-- end repeating kickin part -->
+
                         </div>
+
```

Comment:
totally faked data, but near-final HTML.  will replace with the data populated from the backend soon